src/css/main.css

```diff
@@ -0,0 +1,35 @@
+* {
+    box-sizing: border-box;
+}
+
+a {
+    margin:0;
+    padding:0;
+    font-size:100%;
+    vertical-align:baseline;
+    background:transparent;
+    text-decoration: none;
+    color: inherit;
+}
+
+body * {
+    font-family: 'Open Sans', sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Fira Mono', monospace;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
```

Comment:
This solves #25 since it only hides x-axis overflow instead of both (old line was `overflow: hidden`).